src/components/Home/WorksList/Project/Project.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import data from "../../../../data.json";
+import "./Project.css";
+
+export default function Project(props) {
```

Comment:
this component is a page on its own .. it does not belong to the `worksList` itself per say .. a better structure is to put it in in the components directorty for example , or even better have a pages directory and put the Home and this one in it .. also I would name it `projectPage`.js for example